app/screens/LocationDetails.js

```diff
@@ -102,6 +102,18 @@ class LocationDetails extends Component {
                                         style={{borderRadius: 50}}
                                         containerStyle={{marginTop:20,marginBottom:10,marginRight:20,marginLeft:20,borderRadius:50}}
                                     />
+                                    <Button 
+                                        title={'View My Saved Locations'}
+                                        onPress={() => this.props.navigation.navigate('Saved')}
```

Comment:
@bpoore This doesn't totally work. You need to click the button, and then click the "Great!" button for it to go to the Saved page. I'm not sure how to make it so you the onPress both closes the modal and sends you to Saved.

Reply:
@RyanTG Do you want to go to the saved page here? We can update that, but I didn't realize that was the objective. I'm open to your thoughts, but when clicking to add a location to my favorites, I don't necessarily want to be navigated to the favorites at that time.

Reply:
On the modal that shows “location added to your saved list” and there’s a button that says “Great!” I wanted to add a second button that says “go to my saved locations”. Then, instead of closing the modal and showing the location details, a user could opt to close the modal and view all of their saved locations. Does that make sense?

Reply:
Got it working

Reply:
I deleted my original comment. Apologies just quickly looking over things, I didn't realize there was a new button. It's fixed up now.